### PR TITLE
Expose get_total_visible_rows method to GDScript

### PR DESCRIPTION
**Pull request by sent44**
I attempt to commit it to exist PR, #42522, but I am very confuse as I am new to PR stuff so I open new PR.

### CODE REVIEW

File: doc/classes/TextEdit.xml [hunk 1]
```diff
@@ -282,6 +282,13 @@
 				Returns the selection end line.
 			</description>
 		</method>
+		<method name="get_visible_line_count" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the number of visible lines, including text warped.
```
Comment (KoBeWi) [inline doc/classes/TextEdit.xml:289]:
```suggestion
				Returns the number of visible lines, including wrapped text.
```

Comment (KoBeWi) [reply doc/classes/TextEdit.xml]: You did another type. Should be w**ra**pped not w**ar**pped

Comment (sent44) [reply doc/classes/TextEdit.xml]: Oh, sorry ;-;

**Review by KoBeWi — APPROVED** [review]
Looks ok, but [commits need to be squashed](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase).

**Comment by sent44** [comment]
So I can rebase and force push, right?
I am on it.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**